tests/debugger/test_debugger_base.py

```diff
@@ -2,32 +2,63 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2021 Datadog, Inc.
 
-from utils import scenarios, interfaces, weblog, features, missing_feature, context
+from utils import interfaces
 from utils.tools import logger
-
-
-def validate_probes(expected_probes):
-    def get_probes_map():
-        agent_logs_endpoint_requests = list(interfaces.agent.get_data(path_filters="/api/v2/logs"))
-        probe_hash = {}
-
-        for request in agent_logs_endpoint_requests:
-            content = request["request"]["content"]
-            if content is not None:
-                for content in content:
+from packaging import version
+import json
+import re
```

Comment:
Reorganize the imports: https://peps.python.org/pep-0008/#imports

Reply:
Reorganized here, if I got it right 446c34081c7069016e6f09f27ff6648c10369c00